raiden/utils/migrations/v16_to_v17.py

```diff
@@ -41,12 +41,15 @@ def _transform_snapshot(raw_snapshot):
 
 
 def _transform_snapshots(storage: SQLiteStorage):
+    updated_snapshots_data = []
     for snapshot in storage.get_snapshots():
         new_snapshot = _transform_snapshot(snapshot.data)
-        storage.update_snapshot(snapshot.identifier, new_snapshot)
+        updated_snapshots_data.append((new_snapshot, snapshot.identifier))
```

Comment:
This whole thing can be just a simple list comprehension:

```python
update_snapshots_data = [
    (_transform_snapshot(snapshot.data), snapshot.identifier)
    for snapshot in storage.get_snapshots()
]
```

Reply:
Sure but this is just a stylistic approach so I will leave it as is.